src/main/java/com/gregtechceu/gtceu/api/data/chemical/material/properties/HazardProperty.java

```diff
@@ -92,10 +93,11 @@ public enum ProtectionType{
 
         public boolean isProtected(LivingEntity livingEntity){
             List<ArmorItem.Type> correctArmorItems = new ArrayList<>();
-            for (ArmorItem.Type equipmentType: equipmentTypes){
+            for (ArmorItem.Type equipmentType: equipmentTypes) {
                 ItemStack armor = livingEntity.getItemBySlot(equipmentType.getSlot());
-                if (!armor.isEmpty() && armor.getItem() instanceof ArmorComponentItem armorItem && armorItem.getArmorLogic().isPPE())
-                    correctArmorItems.add(armorItem.getType());
+                if(!armor.isEmpty() && ((armor.getItem() instanceof ArmorComponentItem armorItem && armorItem.getArmorLogic().isPPE()) || armor.getTags().anyMatch(tag -> tag.location().equals(new ResourceLocation("forge", "ppe"))))){
```

Comment:
make the tag:
1. a constant in `CustomTags`
2. you can just do `tag.equals(CustomTags.PPE_ARMOR)`

Reply:
3. it can be a tag in the `gtceu` namespace.